README.md

```diff
@@ -1,23 +1,104 @@
+site, which is fully functional in terms of controlling the user permissions for
+tracking their use of the site via Google Analytics.
+
+Before adding the cookie consent banner, you will need to obtain a Google 
+Analytics key that's both unique to your app and stored within the DfE Google
+Analytics domain. You can request a key from the 
+[Statistics Development team](mailto:statistics.development@education.gov.uk). 
+This should then be added as a variable to your dashboard's 
+[global.R](https://github.com/dfe-analytical-services/dfeshiny/blob/cookie-module/tests/test_dashboard/global.R) 
+file (replacing `ABCDE12345` below with the key provided by the Statistics 
+Development team):
+
+```
+google_analytics_key <- 'ABCDE12345'
+```
+
+Next, you should copy the javascript file
+([cookie_consent.js](https://raw.githubusercontent.com/dfe-analytical-services/dfeshiny/cookie-module/js/cookie-consent.js)) 
+necessary for controlling the cookie storage in the user's browser to the `www/` 
+folder in your dashboards repository.
+
+Once you've made the above updates, add the following lines to your 
+[ui.R](https://github.com/dfe-analytical-services/dfeshiny/blob/cookie-module/tests/test_dashboard/ui.R) 
+script (updating "My DfE R Shiny data dashboard" with the name of your app):
+
+```
+dfe_cookie_script(),
+cookie_banner_ui("cookies", name = "My DfE R Shiny data dashboard"),
+```
+
+Putting these on the lines *just before* the `shinyGovstyle::header(...)` line 
+should work well.
+
+Then add the following code to your
+[server.R](https://github.com/dfe-analytical-services/dfeshiny/blob/cookie-module/tests/test_dashboard/server.R) 
+script somewhere *inside* the `server <- function(input, output, session) {...}` 
+function (you shouldn't need 
+to change anything in this one):
+
+```
+output$cookie_status <- dfeshiny::cookie_banner_server(
+  "cookies",
+  input_cookies = reactive(input$cookies),
+  input_clear = reactive(input$cookie_consent_clear),
+  parent_session = session,
+  google_analytics_key = google_analytics_key
+)
+```
+
+Finally, you should make sure you're using the `dfeshiny::support_panel()` 
+function within the `navListPanel(...)` in your 
+[ui.R](https://github.com/dfe-analytical-services/dfeshiny/blob/cookie-module/tests/test_dashboard/ui.R) 
+script as this will provide
+users with the necessary explanatory text on how we use cookies and the ability 
+to change their decision on whether or not to accept the use of cookies.
+
+### Adding a custom disconect message to your dashboard
+
+dfeshiny provides a function to add a custom disconnect message to your dashboard - this will appear when a dashboard would otherwise 'grey-screen' and will include options to refresh the page, go to overflow sites or visit the publication directly on Explore Education Statistics. 
+
+To include a custom disconnect message, you should insert the following line into the ui.R script: 
+
+```
+customDisconnectMessage(),
```

Comment:
This'll need to be something like:
```
customDisconnectMessage(
   links=c("https://department-for-education.shinyapps.io/dashboard","https://department-for-education.shinyapps.io/dashboard-overflow"), 
   publication_name = 'Publication title', 
   publication_link = 'https://explore-education-statistics.gov.uk/find-statistics/publication-title'
)
```

As in it won't work now without the user passing that information explicitly to the function.